microsoft/testsuites/core/TestScripts/hello.c

```diff
@@ -0,0 +1,5 @@
+#include <stdio.h>
```

Comment:
use lower case for folders, it can be considered as test data, not test script. The c code is not script. So use `test_data` as folder name.

Reply:
updated